plugins/BEdita/API/tests/TestCase/Utility/JsonApiTest.php

```diff
@@ -563,4 +563,44 @@ public function testFallbackLinks()
 
         static::assertEquals($expected, $result);
     }
+
+    /**
+     * Test {@see \BEdita\Core\Utility\JsonApi::formatData()} and
+     * {@see \BEdita\Core\Utility\JsonApi::extractAttributes()} methods.
+     *
+     * @return void
+     *
+     * @covers ::formatData
+     * @covers ::formatItem
+     * @covers ::extractAttributes
+     */
+    public function testFormatDataArray()
```

Comment:
If you choose to follow the advice for `JsonApi` utility class, this test becomes obsolete — and will be the only one to fail in the test suite, which is correct.